### ISSUE TITLE
Unable to sign commits using gpg: "Error: error: cannot run gpg: No such file or directory"

### ISSUE THREAD

**Issue by aystic** [kevinmkchin/Obsidian-GitHub-Sync#7]
<img width="413" alt="SCR-20240430-pcog" src="https://github.com/kevinmkchin/Obsidian-GitHub-Sync/assets/54324845/fed8a0e0-79f8-40b8-86f3-98c3deee9a7e">

**Comment by dedestem** [comment]
Why? Why do you even need this? You keep it in an private repo?

**Comment by aystic** [comment]
You are correct. I don't need GPG signing in a private repo. I have GPG signing enabled globally for all repos by default. I didn't know that it doesn't work with the plugin so when I tried to use the plugin without disabling it for this repo, I got the errors.

**Comment by Falci** [comment]
```
git config commit.gpgsign false
```

**Comment by sixtop** [comment]
Had same error, fixed by adding additional PATH environment of my gpg implementation in obsidian git plugin settings.

```
/opt/homebrew/bin/
```
